src/env.h

```diff
@@ -263,6 +265,9 @@ class Environment {
     enum Fields {
       // Set this to not zero if the init hook should be called.
       kCallInitHook,
+      // Set this to not zero if async wrap should be enabled. This is
+      // automatically set when SetupHooks is called.
+      kEnabled,
```

Comment:
No. `kEnable` is what "enables" `init` / `before` / `after` to be called. But `init` is only called if `kCallInitHook` is true.